### DISCUSSION TITLE
Downside to install firewalld gui???

### DISCUSSION THREAD

**Discussion by FritzLott** [Security-Onion-Solutions/securityonion#5640] [Unsupported Versions] [answered]
As a Linux newbie it would be nice to make firewall changes with either CLI or GUI.  Is there some (hidden) downside to installing the Firewalld GUI on a Security Onion standalone installation (with the Analyst VM already installed)?

**Comment by FritzLott** [comment]
The difficulty is that firewall-config is not installed as a part of Security Onion.  It is installed in the CentOS 7 laptop I built as a learning device.  Is there some simple way to add firewall-config to Security Onion?  Is it missing because it is some security risk?

**Comment by n0th1nt0lOOz** [comment]
you need a GUI for OS first to use firewall GUI version. SO only provide CLI environment so you can use the firewalld CLI only

**Comment by FritzLott** [comment, marked as answer]
The VM Analyst installs Gnome so a GUI is already available.

After thrashing around quite awhile without success, the solution was so simple:

sudo yum install firewall-config

sudo firewall-config

**Comment by dougburks** [comment]
Please note that using a firewall tool like firewall-config on Security Onion 2 is untested and unsupported. Our firewall rules are managed by salt and if you manually modify using another tool you might end up with unintended consequences. We recommend the following:
https://docs.securityonion.net/en/2.3/so-allow.html#so-allow
https://docs.securityonion.net/en/2.3/firewall.html

**Reply by FritzLott** [reply]
Thank you! Thank you! Just what I needed to know.

I just wanted to use firewall-config as a confirmation of changes to the firewall, not as the primary way of making changes.
